guides/doc-Upgrading_and_Updating/topics/upgrading_satellite_server_parent.adoc

```diff
@@ -1,5 +1,4 @@
-[[upgrading_satellite_server_parent]]
-
+[id="Upgrading_Server_{context}"]
```

Comment:
👍

Reply:
I merged this, but according to modularization standards, the file ID and the file name should be identical... but we're not really there yet with this guide, so I turned a blind eye... and the suggestion is an improvement according to our upstream goals... ;-)

Reply:
I intended to get this branch into accord with master at this point. Once we begin actual modularization with correct prefixed filenames, we will change it accordingly.